lib/scan.ts

```diff
@@ -67,28 +74,59 @@ export async function scan(
   const dockerfileAnalysis = await readDockerfileAndAnalyse(dockerfilePath);
 
   const imageType = getImageType(targetImage);
+  return {
+    targetImage,
+    imageType,
+    dockerfileAnalysis,
+    options,
+  };
+}
+
+export async function scan(
+  options?: Partial<PluginOptions>,
+): Promise<PluginResponse> {
+  const {
+    targetImage,
+    imageType,
+    dockerfileAnalysis,
+    options: updatedOptions,
```

Comment:
why not leave it as options 🙃

Reply:
Because `options` is a parameter to the function and can't be shadowed this way